html/changelog.html

```diff
@@ -53,6 +53,44 @@
 		
 		-->
 		<div class="commit sansserif">
+			<h2 class="date">17 April 2019</h2>
+			<h3 class="author">Anewbe updated:</h3>
+			<ul class="changes bgimages16">
+				<li class="rscadd">The Subspace Jammer traitor item now lasts about 10 minutes. It now blocks signalers from sending or receiving, and also obscures suit sensors around it.</li>
+			</ul>
+			<h3 class="author">N3X15 (MistyLuminescence) updated:</h3>

```

Comment:
:thinking: ?

Reply:
Misty missed putting their name in correctly